### PR TITLE
#164798206 Users should rate articles 

### PR DESCRIPTION
**Pull request by dannylwe**
#### What does this PR do?
This PR enables a user to rate an article, between the range of 0-5

#### Description of Task to be completed?
A user can leave a rating on an article but not more than once

#### How should this be manually tested?
run `git fetch origin users-should-rate-articles-164798206`
checkout `git checkout users-should-rate-articles-164798206`
enter shell `pipenv shell`
run migrations `python manage.py migrate`
runserver `python manage.py runserver`

#### Any background context you want to provide?
To rate an article go to the endpoint `articles/<slug>/rate` and send rating as a JSON object
`{"ratings": <0 - 5>}`

#### What are the relevant pivotal tracker stories?
[#164798206](https://www.pivotaltracker.com/story/show/164798206)

#### Postman collection
[Run in Postman](https://app.getpostman.com/run-collection/f17327eecaa858764bb6)

### CODE REVIEW

File: authors/apps/articles/migrations/0002_auto_20190415_1518.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# Generated by Django 2.2 on 2019-04-11 13:57
+# Generated by Django 2.2 on 2019-04-15 15:18
```
Comment (ja-odur) [inline authors/apps/articles/migrations/0002_auto_20190415_1518.py]: @dannylwe do not merge migrations

File: authors/apps/articles/migrations/0003_article_favorite.py [hunk 2]
```diff
@@ -1,20 +0,0 @@
-# Generated by Django 2.2 on 2019-04-11 15:28
-
-from django.conf import settings
-from django.db import migrations, models
```
Comment (ja-odur) [inline authors/apps/articles/migrations/0003_article_favorite.py]: @dannylwe dont  delete migrations

File: authors/apps/articles/migrations/0002_auto_20190415_1518.py (hunk 1 above)
Comment (dannylwe) [reply authors/apps/articles/migrations/0002_auto_20190415_1518.py]: @ja-odur hi, it looks like I will have to manually copy over the migrations into empty files. I tried and not figured out another efficient way to do so.

File: authors/apps/articles/views.py [hunk 3]
```diff
@@ -244,3 +244,23 @@ def post(self, request, *args, **kwargs):
         article.save()
         serializer = self.serializer_class(article, context={'request':self.request})
         return Response(serializer.data, status=returned_status)
+
+class RatingsView(generics.ListCreateAPIView):
+    """
+    view for rating for articles
+    """
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+    serializer_class = serializers.ArticleRatingSerializer
```
Comment (ja-odur) [inline authors/apps/articles/views.py:253]: @dannylwe Please add a queryset to this view inorder for the get method to work.

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by ezrogha — APPROVED** [review]
LGTM

**Review by felixkiryowa — APPROVED** [review]

**Pull request merged**